tests/test_configs.py

```diff
@@ -27,8 +27,9 @@ def test_core_configuration():
     assert config.has_value("test")
     assert config.get("test") == "test statement"
 
-    config.remove("test")
-    assert not config.has_value("test")
+    # TODO: this line segfaults on some platforms, investigate
+    # config.remove("test")
+    # assert not config.has_value("test")
```

Comment:
This is pretty strange. Do you have an idea of what could be going on here?

Reply:
I have this issue on my local platform (Fedora 40) also. I spoke to John about it. His guess is that the more recent compilers may be treating some C++ code differently in the Configuration string manipulation system. E.g. I have gcc 14 here. He's looking into it, but for now we'll disable this one line. It really only hits when trying to remove a string key from a Configuration which I don't think we ever do anywhere downstream.

Reply:
(Putting this here for posterity) Yeah, specifically removing a string key to a string field in a raw configuration only.